packages/braid-design-system/src/lib/components/TooltipRenderer/TooltipRenderer.tsx

```diff
@@ -125,6 +141,12 @@ export const TooltipRenderer = ({
   const [controlledVisible, setControlledVisible] = useState(false);
   const [opacity, setOpacity] = useState<0 | 100>(0);
   const { grid, space } = useSpace();
+  const triggerBoundingBoxRef = useRef<ReturnType<typeof normaliseRect>>(
+    normaliseRect(),
+  );
+  const tooltipBoundingRectRef = useRef<ReturnType<typeof normaliseRect>>(
+    normaliseRect(),
+  );
```

Comment:
This means `normaliseRect()` will be called (twice) on every render.

Reply:
I'll create a default rect statically outside the component and use that instead.